devtools/conda-envs/test_env.yaml

```
@@ -5,12 +5,12 @@ channels:
 dependencies:
     # Core
   - python
-  - pydantic >=1.10.17,<3
+  - pydantic =2
   - jinja2
 
     # OpenFF stack
   - openff-toolkit-base >=0.14.4
-  - openff-utilities >=0.1.9
+  - openff-interchange =0.4
```

I don't give a lot of thought to this repo, so I'll forget that this pin is here. If interchange 0.5 breaks fragmenter I'd rather see the CI go red on the status repo and decide on a course of action then, since the alternative is not knowing that something is broken. 

```suggestion
  - openff-interchange >=0.4
```